website/README.md

```diff
@@ -55,10 +56,10 @@ Before opening a pull request or push to git repo, you can preview changes from
 
 ## Pushing to site (for committer only)  
 1. `cd website/_site`
-2. `svn status`
-3. You'll need to `svn add` any new files
-4. `svn commit -m 'UPDATE MESSAGE'`
+2. `git status`
+3. You'll need to `git add` any new files
+4. `git commit -m 'UPDATE MESSAGE'`
 
-Within a few minutes, svnpubsub should kick in and you'll be able to
+Within a few minutes, gitpubsub should kick in and you'll be able to
```

Comment:
Hi @martin-g , If you only use `git` to update the website content to the `https://github.com/apache/kylin`, you cannot directly update the kylin website. Because the code of kylin website is actually placed in SVN, and GitHub repo is just a transit station, it still needs to be updated to SVN in order to really update the `http://kylin.apache.org/`, so I don't think this part of the document content needs to be changed.